draft-keranen-t2trg-rest-iot.md

```diff
@@ -527,7 +529,8 @@ The knowledge about these identifiers as well as matching implementations have t
 A client begins interacting with an application through a GET request on an entry point URI.
 The entry point URI is the only URI a client is expected to know before interacting with an application.
 From there, the client is expected to make all requests by following links and submitting forms that are provided in previous responses.
-The entry point IRI can be obtained, for example, by manual configuration or some discovery process (e.g., DNS-SD {{RFC6763}} or Resource Directory {{I-D.ietf-core-resource-directory}}).
+The entry point URI can be obtained, for example, by manual configuration or some discovery process (e.g., DNS-SD {{RFC6763}} or Resource Directory {{I-D.ietf-core-resource-directory}}).
+For Constrained RESTful environments "/.well-known/core" relative URI is defined as a default entry point for requesting the links hosted by a server {{RFC6690}}.
```

Comment:
In this case, your entry point URI is the address of the device hosting the /.well-known/core resource... You have to know or discovery this first...

Reply:
Clarified that knowledge of the address is needed for .w-k